ambry-frontend/src/main/java/com.github.ambry.frontend/AmbryBlobStorageService.java

```diff
@@ -397,14 +413,25 @@ public void onCompletion(String result, Exception exception) {
           logger.trace("Forwarding {} of {} to the router", restMethod, result);
```

Comment:
This message is not valid for getReplicas, but maybe it's not worth changing, since it's a trace message